docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -0,0 +1,158 @@
+
+Example:
+```
+accept: application/expo+json, application/json
+expo-platform: string
+expo-runtime-version: string
+```
+
+## Manifest Response
+
+A conformant server MUST return a response containing a [manifest body](#manifest-response-body) and [headers](#manifest-response-headers).
+
+The choice of manifest and headers are dependent on the values of the request headers. A conformant server MUST:
+
+* Return a manifest that describes the most recent update ordered by creation time satisfying all constraints imposed by the [request headers](#manifest-request).
+
+### Manifest Response Headers
+
+```
+expo-protocol-version: 0
+expo-sfv-version: 0
+expo-manifest-filters: &lt;expo-sfv&gt;
+expo-server-defined-headers: &lt;expo-sfv&gt;
+cache-control: &lt;*&gt;
+content-type: &lt;*&gt;
+```
+
+* `expo-protocol-version` describes the version of the protocol defined in this spec and MUST be `0`.
+* `expo-sfv-version`  MUST be `0`.
+* `expo-manifest-filters` is an [Expo SFV 0](expo-sfv-0.md) dictionary. It is used to filter updates stored by the client library by the `metadata` attribute found in the [manifest](#manifest-response-body). If a field is mentioned in the filter, the corresponding field in the metadata must either be missing or equal for it to pass. The client library MUST store the manifest filters until it is overwritten by a newer response.
+* `expo-server-defined-headers` is an [Expo SFV](expo-sfv.md) dictionary. It defines headers that a client library MUST store this, until it is overwritten by a newer response, and include it in every subsequent [manifest request](#manifest-request).
+* `cache-control` - A value of `cache-control: private, max-age=0` is recommended to ensure the newest manifest is returned. Setting longer cache ages could result in stale updates.
+* `content-type` MUST be determined by _proactive negotiation_ as defined in [RFC 7231](https://tools.ietf.org/html/rfc7231#section-3.4.1). Since the client library is [required](#manifest-request) to send an `accept` header with each manifest request, this will always be either `application/expo+json`, `application/json`, or a `406`.
+
+
+The manifest endpoint MUST also be served with a `cache-control` header set to an appropriately short period of time. For example:
+
+```
+cache-control: max-age=0, private
+```
+
+### Manifest Response Body
+
+The body of the response MUST be a manifest, which is defined as JSON conforming to the following structure:
+```
+{
+  id: &lt;string&gt;
+  createdAt: &lt;datetime&gt;
+  runtimeVersion: &lt;string&gt;
+  launchAsset: Asset
+  assets: Asset[]
+  metadata: { &lt;string&gt;:&lt;string&gt; }
+}
+```
+_Asset_ is defined as JSON with the following structure:
+```
+{
+  hash: &lt;string&gt;
+  key: &lt;string&gt;
+  contentType: &lt;string&gt;
+  url: &lt;string&gt;
+}
+```
```

Comment:
Curious in general what the correct syntax to use here is. Made a best guess following some comments by @ide.

also `<` and `>` don't seem to display consistently when running locally so switch to `&lt;`,`&gt;`.

Reply:
I think what's tricky is that there isn't a standard syntax for demonstrating types. We could use TypeScript as the closest match. What do you think?